utils/monitoring.js

```diff
@@ -0,0 +1,113 @@
+const { errors } = require('arsenal');
+const promClient = require('prom-client');
+const { Registry } = require('prom-client');
+const CountMaster = require('../CountItems/CountMaster');
+
+const aggregatorRegistry = new promClient.AggregatorRegistry();
+const { collectDefaultMetrics } = promClient;
+
+const bucketCount = (promClient.register.getSingleMetric('bucketCount_metrics'))
+    || new promClient.Gauge({
+        name: 'bucketCount_metrics',
```

Comment:
Need to follow naming conventions for metrics:
* s3 prefix
* name of component
* lowercase
* end with an indication of the unit

Also this metric is indeed the _"total number fo buckets processed"_, meaning:
- it is not a gauge, but really a counter (even increasing!)... and it is up to prometheus to "filter out" resets of the metric
- the name must clearly convey the fact this is not the current number of buckets...

```suggestion
        name: 's3_countitems_total_buckets_count',
```

Reply:
removed as this metric can be retrieved based on the comment here : https://github.com/scality/s3utils/pull/312#discussion_r1601020512